Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -23,34 +23,53 @@ private static void PromptPersonInformation(string person)
             float height = PromptFloatInput("Please provide your height in m: ");
 
             Console.WriteLine(name + " " + surname + " is " + age + " years old, their weight is " + weight + " kg and their height is " + height + "cm.");
-            float bmi = CalculateBmi(weight, height);
+            float bmi = GetBmi(weight, height);
             Console.WriteLine("Their BMI is: " + bmi);
         }
 
-        public static float CalculateBmi(float weight, float height)
+        public static float GetBmi(float weight, float height)
+        {
+            bool areParametersValid = ValidateWeightAndHeight(weight, height);
+            if (!areParametersValid)
+            {
+                return -1;
+            }
+            else 
+            {
+                return CalculateBmi(weight, height);
+            }
+            
+        }
+
+        private static float CalculateBmi(float weight, float height)
+        {
+            float bmi = weight / (height * height);
+            return bmi;
+        }
+
+        private static bool ValidateWeightAndHeight(float weight, float height)
```

Comment:
You should not validate seperately, do it in the same function where you get weight, in another function where you get height.
This function should be removed.